### PR TITLE
ARSSTB-663 Welsh Date Formats + Even More Welsh Changes

### PR DESCRIPTION
**Pull request by Ewan-James-Donovan**
### Type of change
- [ ] Breaking change
- [x] Non-breaking change
- [x] Cosmetic change

### Description
Jira link: [Jira title](https://jira.tools.tax.service.gov.uk/browse/ARSSTB-663)

### CODE REVIEW

File: app/models/ApplicationForAccountHome.scala [hunk 1]
```diff
@@ -34,8 +35,12 @@ final case class ApplicationForAccountHome(
...
   actions: Seq[ActionItem]
 ) {
-  def dateString(): String =
-    ApplicationForAccountHome.formatter.format(date)
+  def dateString(lang: Lang = Lang("en")): String =
+    if (lang == Lang("en")) {
+      ApplicationForAccountHome.formatter.format(date)
+    } else {
+      ApplicationForAccountHome.welshFormatter.format(date)
+    }
```
Comment (aaocs) [inline app/models/ApplicationForAccountHome.scala:43]: I'd do this the other way round so the default (else) is English.

File: worksheet.sc
Comment (aaocs) [inline worksheet.sc:1]: Delete this file

File: app/models/ApplicationForAccountHome.scala (hunk 1 above)
Comment (aaocs) [inline app/models/ApplicationForAccountHome.scala:43]: I'd have the default (else) as English, but not a major issue.

**Review by aaocs — CHANGES_REQUESTED** [review]
Delete worksheet.sc

File: app/models/ApplicationForAccountHome.scala (hunk 1 above)
Comment (Ewan-James-Donovan) [reply app/models/ApplicationForAccountHome.scala:43]: the default case is english:  def dateString(lang: Lang = Lang("en")):

Comment (Ewan-James-Donovan) [reply app/models/ApplicationForAccountHome.scala:43]: it is the dfeault case def dateString(lang: Lang = Lang("en")):

**Pull request closed without merging**